cap-ci/pipeline.template

```diff
@@ -1139,6 +1147,54 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $stratos }}
+{{ end }}
+
+{{- if index $config.jobs "clean-kubecf" }}
+- name: clean-kubecf-{{ $scheduler }}-{{ $backend }}-{{ $option }}
```

Comment:
should be a cleanup job for all deployments

Reply:
not sure if I understand this?

Reply:
make stratos-clean
make metrics-clean

Reply:
I guess this is being handled now.